.travis.yml

```diff
@@ -24,6 +24,7 @@ env:
  - CABALVER=1.18 GHCVER=7.8.4
  - CABALVER=1.22 GHCVER=7.10.3
  - CABALVER=1.24 GHCVER=8.0.1
+ - CABALVER=1.24 GHCVER=8.2.1
```

Comment:
why not CABALVER=2.0?

Reply:
You're right it should use Cabal 2.0. I've created a PR to fix that up.
Apologies.